modules/core/02-client/types/client.go

```diff
@@ -71,6 +72,14 @@ func (ics IdentifiedClientStates) Sort() IdentifiedClientStates {
 	return ics
 }
 
+// NewLiteCounterparty creates a new LiteCounterparty instance
+func NewLiteCounterparty(clientID string, merklePathPrefix *commitmenttypes.MerklePath) LiteCounterparty {
```

Comment:
I think we can just remove the `Lite` in this name completely and just call it the `Counterparty` of the Client